librosa/beat.py

```diff
@@ -226,13 +226,13 @@ def tempo(
 
     Parameters
     ----------
-    y : np.ndarray [shape=(n,)] or None
-        audio time series
+    y : np.ndarray [shape=(..., n)] or None
```

Comment:
I can see audio time-series being of shapes `[shape=(n,)] ` and `[shape=(m, n)] `. But what does it mean for "audio time-series to be `[shape=(..., n)] ` :thinking:

Reply:
... in numpy means 0 or more dimensions.  My plan is to put together an explainer section / advanced example notebook in the docs to explain this convention, because otherwise we'd have the same description sprinkled everywhere in the docstrings.

As for why ... instead of `m`, one use-case could be a 2d grid microphone array, where you might want to have `shape=(n_x, n_y, t)` (something like in the isophonics RIR set).

Reply:
Ok. I'm sure it could be useful in other ways, but "flat" 2d grid microphone arrays are pretty rare, and most microphone array datasets in A-format are loaded as a matrix where rows are the channels in a pre-determined (often custom) order and the columns are the samples.

Reply:
Agreed; maybe a better example would be a mini-batch of stereo signals: `(n_batch, 2, n_samples)`.  But the point is that we don't actually have or need any restrictions on leading dimensions most of the time.